java.lang.NoClassDefFoundError: com/palantir/roboslack/clients/SlackClients when depending on roboslack-webhook 1.1.0

Try depending on `roboslack-webhook` 1.1.0 and running this sample code:

```
public static void main(String[] args) {

  WebHookToken token = WebHookToken.fromEnvironment();

 MessageRequest msg = MessageRequest.builder()
                .username("Slack Oncall Service")
                .iconEmoji(SlackMarkdown.EMOJI.decorate("sos"))
                .channel("slack-dev-msgs")
                .text("[CRITICAL] Error Logged in MailService - " + SlackMarkdown.LINK.decorate(url
                        ("https://localhost:9600/?pretty"), "[Logstash Link]"))
                .build();
        ResponseCode code = SlackWebHookService.with(token).sendMessage(msg);

        System.out.println(code);

}
```

We receive this error message:

```
Exception in thread "main" java.lang.NoClassDefFoundError: com/palantir/roboslack/clients/SlackClients
	at com.palantir.roboslack.webhook.SlackWebHookService.<init>(SlackWebHookService.java:53)
	at com.palantir.roboslack.webhook.SlackWebHookService.with(SlackWebHookService.java:64)
	at Main.main(Main.java:58)
Caused by: java.lang.ClassNotFoundException: com.palantir.roboslack.clients.SlackClients
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 3 more
```

--- a/extras/jackson-support/build.gradle
+++ b/extras/jackson-support/build.gradle
@@ -16,6 +16,7 @@
 
 apply from: "${rootDir}/gradle/java.gradle"
 apply from: "${rootDir}/gradle/junit.gradle"
+apply from: "${rootDir}/gradle/publish.gradle"
 
 dependencies {
     compile "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"

--- a/extras/slack-clients/build.gradle
+++ b/extras/slack-clients/build.gradle
@@ -16,6 +16,7 @@
 
 apply from: "${rootDir}/gradle/java.gradle"
 apply from: "${rootDir}/gradle/junit.gradle"
+apply from: "${rootDir}/gradle/publish.gradle"
 
 dependencies {
     compile project(":extras:jackson-support")

